spring-mvc-basics-4/src/main/java/com/baeldung/jsonparam/ParamObjectDTO.java

```diff
@@ -0,0 +1,43 @@
+package com.baeldung.jsonparam;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ParamObjectDTO {
```

Comment:
Let's give this domain object a better name, I'm not sure what it is supposed to represent? Your controller has /message in the URL but this class doesn't seem to represent a message. It is confusing, at least to me.